### ISSUE TITLE
rprint and IO

### ISSUE THREAD

**Issue by simonbyrne** [JuliaStats/RCall.jl#54]
It would be useful to be able to `rprint` to different IO objects (at the very least, we could then define nicer `show` methods for SEXPREC objects).

There doesn't seem to be a convenient way to get this directly from R, so I guess we have to use `redirect_stdout`?

**Comment by randy3k** [comment]
I have implemented something similar in the RCalling.jl. Check https://github.com/randy3k/RCalling.jl/blob/master/src/sexp.jl#L28

**Comment by simonbyrne** [comment]
Ah, great thanks!

**Comment by randy3k** [comment]
Just one more thing, my code does not capture STDERR, we should also do `redirect_stderr`!?

**Comment by simonbyrne** [comment]
On my machine they appear at the end of the Julia session.
```
julia> using RCall

julia> rcall(:warning,"hello")
StrSxp(536870992,Ptr{Void} @0x00007fb514770978,Ptr{Void} @0x00007fb5191af5c8,Ptr{Ptr{None}} @0x00007fb5191af5f0,1,0)

julia>  quit()
Warning message:
hello 
```

**Comment by simonbyrne** [comment]
I've changed the IO, and fixed the warning problem in #59

**Comment by simonbyrne** [comment]
Looks like i was too ambitious: `Rf_PrintWarnings` doesn't work on linux (it's an unsupported function, but seems to work fine on Windows and OS X). Apparently we're [not the only ones](https://stat.ethz.ch/pipermail/r-devel/2013-March/066058.html) who want this, but it doesn't look like the r-dev team are in a hurry to fix it.
